stunnel/stunnel.ml

```diff
@@ -156,14 +157,8 @@ let config_file verify_cert extended_diagnosis host port legacy =
 			[]
 	) @ (
 		if legacy then [
-			"sslVersion = all";
-			"options = NO_SSLv2";
 			"options = NO_SSLv3";
-			"ciphers = " ^ (good_ciphers ()) ^ (match !legacy_ciphersuites with "" -> "" | s -> (":" ^ s))
-		] else [
-			"sslVersion = TLSv1.2";
-			"ciphers = " ^ (good_ciphers ());
-		]
+        ] else []
 	)
   in
     String.concat "" (List.map (fun x -> x ^ "\n") lines)
```

Comment:
This is the same as `String.concat "\n" lines` except that there would be no newlines at the end, which isn't needed anyway. I understand that this is existing code and you might want to keep it the way it is because we are backporting here.